### PR TITLE
Update CODEOWNERS to have reviewer group always

### PR DESCRIPTION
**Pull request by benjdlambert**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by freben — APPROVED** [review]

**Review by Rugvip — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,50 +4,50 @@
...
+/plugins/azure-devops                               @backstage/reviewers @marleypowell @awanlin
+/plugins/azure-devops-backend                       @backstage/reviewers @marleypowell @awanlin
+/plugins/azure-devops-common                        @backstage/reviewers @marleypowell @awanlin
+/plugins/bitrise                                    @backstage/reviewers @backstage/sda-se-reviewers
+/plugins/catalog-graph                              @backstage/reviewers @backstage/sda-se-reviewers
+/plugins/circleci                                   @backstage/reviewers @adamdmharvey
+/plugins/cloudbuild                                 @backstage/reviewers @trivago/ebarrios
+/plugins/code-coverage                              @backstage/reviewers @alde @nissayeva
+/plugins/code-coverage-backend                      @backstage/reviewers @alde @nissayeva
+/plugins/cost-insights                              @backstage/reviewers @backstage/silver-lining
```
[the remark this answers is not in the document]
Comment (benjdlambert) [reply .github/CODEOWNERS]: no?

[the remark this answers is not in the document]
Comment (freben) [reply .github/CODEOWNERS]: i think this is where the extra space was :)

[the remark this answers is not in the document]
Comment (Rugvip) [reply .github/CODEOWNERS]: no - yes

[the remark this answers is not in the document]
Comment (benjdlambert) [reply .github/CODEOWNERS]: ah ok - fixed.

**Pull request merged**